fr/index.html

```diff
@@ -17,22 +17,22 @@
     <h1>Ne demande pas la permission de demander, demande juste.</h1>
     <p>
       De temps en temps, dans des channels que je fréquente,
-      je vois quelqu'un apparaître et demander,
+      quelqu'un arrive et dit :
```

Comment:
"someone pops in" was literally translated to "I see someone appearing". "[to pop in](https://www.wordreference.com/enfr/pop%20in)" is an english-specific idiom.

I replaced it with a shorter, more straightforward translation : "quelqu'un arrive" which would translate literally to "someone arrives [in the channel]"